.travis.yml

```diff
@@ -22,13 +22,13 @@ install:
  - travis_retry nix-channel --update
 
 script:
- - outs=$(nix-build buildable.nix) && echo Produced $outs
+ - nix-build ci.nix -A buildOutputs
```

Comment:
any way default.nix could be made the only entry-point?

Reply:
You mean like having a `ci` attribute in `default.nix` and do `nix-build -A ci.buildOutputs` here?

Reply:
yeah. I'm not sure about it but it's a bit tricky when there are multiple end-points. For example just today in one of my projects, I forgot to update the `shell.nix` `{ nixpkgs }` reference when I had changed it in `default.nix`

Reply:
It probably would work to have it in `default.nix`, you'd just need some form of self reference for the new `ci` attribute to access the packages. Or simply do `ci = import ./ci.nix { inherit pkgs }` and add "ci" as a reserved name.

But I think working on that goes outside the scope of this PR.